### DISCUSSION TITLE
new version?

### DISCUSSION THREAD

**Discussion by schmitch** [UglyToad/PdfPig#349] [General]
will there be a new version, soon?

**Comment by EliotJones** [comment]
Hi @schmitch to be honest I'm suffering quite severe maintainer burnout at the moment. I'd like to fix a few bugs and get the big PR in before making a new version but at the moment the thought of opening the codebase makes me want to curl into a ball and drink!

I'm hoping to be back to progress soon but it's going to take some time and it's already taken longer than I anticipated.

**Reply by schmitch** [reply]
No Problem, take your time and dont spend too much in the project. Its important to also take time off the computer!

**Reply by Poltuu** [reply]
Hi @EliotJones . My company (https://github.com/LuccaSA) has adopted this great project, and we are now processing millions of pdf per month. We are really enthusiastic about this library, and we are willing to invest some time to help, especially regarding the release aspect, for instance to allow nightly builds, or weekly alphas, so no one has to curl into a ball anymore :D !

Right now, we have to use a fork to be able to use the latest fixes quickly, but that's not a very sexy solution.

Let me know if you're interested, and how we could help on that front.

**Reply by EliotJones** [reply]
Hi @Poltuu good to hear, thanks for the offer. I had actually considered using something like CodeShelter or similar to protect the project from a single point of failure, e.g. my getting hit by a bus. More maintainers is always good.

I added a NuGet API key to the repository's secrets (valid for the next year) and a script to generate and push a nightly build but since YAML and devops are activities I absolutely loathe I think I'll need some help to get a working configuration. https://github.com/UglyToad/PdfPig/blob/master/.github/workflows/nightly_release.yml

**Reply by EliotJones** [reply]
Versioning detection is crapping out and I had to trigger the build manually still and the action failed but the first nightly build is pushed https://www.nuget.org/packages/PdfPig/0.1.5-alpha-20210817-b1f88

Only took 12 revisions to get something that did anything, I %£%£$!ing hate DevOps.

**Reply by cremor** [reply]
@EliotJones Thanks! One problem though: NuGet (both the website and the package manager in Visual Studio) see the new nightly build package with a lower version number than the previous alpha builds. Could you please increase the version number so that future nightly builds are shown as updates?

**Reply by Poltuu** [reply]
@EliotJones Thank you for your sacrifice 🙏 I saw the struggle and felt bad for you :$

**Reply by EliotJones** [reply]
> 
> 
> @EliotJones Thanks! One problem though: NuGet (both the website and the package manager in Visual Studio) see the new nightly build package with a lower version number than the previous alpha builds. Could you please increase the version number so that future nightly builds are shown as updates?

@cremor this will be fixed for all post 0.1.6 versiosn, I'm not going to do anything for the 0.1.5 versions since this is just a small issue from the switch of manual to nightly builds. Once 0.1.6 is out the versions will be presented correctly by NuGet.

**Reply by cremor** [reply]
@EliotJones Ok, understandable - assuming you mean "post 0.1.5", since those then will be 0.1.6 alphas, right?
But then I hope it doesn't take another 9 months until 0.1.5 is released 😉

**Reply by fviollette** [reply]
Hi @EliotJones,

> Versioning detection is crapping out and I had to trigger the build manually still and the action failed but the first nightly build is pushed https://www.nuget.org/packages/PdfPig/0.1.5-alpha-20210817-b1f88

Could you please trigger a new build in order to have another new nightly build ? 

Thanks !

**Reply by Poltuu** [reply]
@EliotJones Thank you thank you for your effort on the nightly build 🙏 🙏 🙏 , we all can tell it was a tough one :)
